src/scss/_footer.scss

```diff
@@ -0,0 +1,106 @@
+//---------------Break Points---------------//
+//50em (800px)
+//56.25em (900px)
+//90em (1440px)
+
```

Comment:
You can put these in a sass variable, so `$bp-small: 50em;`, then call the variable like you do colors.